stix_shifter_modules/alertflex/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,48 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "src_ip": ["a.srcIp"],
+      "dst_ip": ["a.dstIp"],
+      "ip": ["a.dstIp", "a.srcIp"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["a.srcPort"],
+      "dst_port": ["a.dstPort"]
+    }
+  },
+  "file": {
+    "fields": {
+      "name": ["a.fileName"],
+      "hashes.SHA-256": ["a.hashSha256"],
+      "hashes.SHA-1": ["a.hashSha1"],
+      "hashes.MD5": ["a.hashMd5"]
+    }
+  },
+  "process": {
+    "fields": {
+      "name": ["a.processName"],
+      "pid": ["a.processId"]
+    }
+  },
+  "user": {
```

Comment:
This would either need to be treated as a custom object (as `x-alertflex-user`) or as the `user-account` object in which case the property would not be `name` but instead be `user_id`